public/acc_tcga_pan_can_atlas_2018/meta_study.txt

```diff
@@ -3,4 +3,5 @@ cancer_study_identifier: acc_tcga_pan_can_atlas_2018
 name: Adrenocortical Carcinoma (TCGA, PanCancer Atlas)
 short_name: ACC (TCGA PanCan)
 description: Adrenocortical Carcinoma TCGA PanCancer data. The original data is <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">here</a>. The publications are <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">here</a>.
-groups: PUBLIC;PANCAN
\ No newline at end of file
+groups: PUBLIC;PANCAN
+pmid: 29625048, 29596782, 29622463, 29617662, 29625055, 29625050
```

Comment:
I think we also need a line `citation: TCGA, Cell 2018`

Reply:
@jjgao just updated.